examples/apexlog/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/airbrake/gobrake/v5/apexlog"
+	"github.com/apex/log"
+)
+
+func main() {
+	// Initialize the airbrake notifier
+	airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+		ProjectId:   363389,                             // use it as an environment variable
+		ProjectKey:  "baa755018d5e35e07897ee0087fcce9c", // use it as an environment variable
+		Environment: "production",                       // use it as an environment variable
+	})
+
+	// Close the airbrake notifier
+	defer airbrake.Close()
+
+	// Create the airbrake handler for "github.com/apex/log"
+	// Insert the airbrake notifier instance and set the severity level.
+	// The acceptable severity levels are "debug" "info", "warning", "error", and "fatal".
+	// If the severity level is not recognized, "error" severity is used.
+	airbrakeHandler := apexlog.New(airbrake, "error")
+
+	log.SetLevel(log.DebugLevel)
+
+	// Set the airbrake handler in the log
+	log.SetHandler(airbrakeHandler)
+
+	ctx := log.WithFields(log.Fields{
+		"file": "something.png",
+		"type": "image/png",
+		"user": "tobi",
+	})
+
+	go func() {
+		for range time.Tick(time.Second) {
+			ctx.Debug("doing stuff")
+		}
+	}()
+
+	go func() {
+		for range time.Tick(100 * time.Millisecond) {
+			ctx.Info("uploading")
+			ctx.Info("upload complete")
+		}
+	}()
+
+	go func() {
+		for range time.Tick(time.Second) {
+			ctx.Warn("upload slow")
+		}
+	}()
+
+	go func() {
+		for range time.Tick(2 * time.Second) {
+			err := errors.New("boom")
+			ctx.WithError(err).Error("upload failed")
+		}
+	}()
+
+	select {}
```

Comment:
This example looks rather complex IMO. What we could do is to:

1. Set severity to `warning`
2. Send a debug log and explain it won't be delivered
3. Send a fatal log and explain that it will be delivered
4. Set severity to `debug`
5. Send a debug log and explain that it will be delivered

No need for goroutines.